ui/src/components/settings/sub_pages/profile/mod.rs

```diff
@@ -69,6 +68,42 @@ pub fn ProfileSettings(cx: Scope) -> Element {
         image.eq("\0") || image.is_empty() || identity.contains_default_picture();
     let no_banner_picture = banner.eq("\0") || banner.is_empty();
 
+    let seed_phrase: &UseState<Option<String>> = use_state(cx, || None);
```

Comment:
I wonder if we should temporarily store it here (eg maybe have a task with a timer to wipe this out after a specific duration, and maybe add `zeroize` to it for more measure, though this crate isnt needed) so it doesnt somehow leak out in the client after it is moved from a more secured space to an unsecured space.

Reply:
@dariusc93 I was hoping after this goes in you and I can take another final pass at this issue and add the feature to disable the storing of the phrase as well as other security measures like having it be removed from memory after some time. Can we get this one in then spend some time today on this?